docs/deploy.rst

```diff
@@ -227,12 +222,6 @@ monitored by the development team and MozMEAO.
     git push
     cd ..
 
-* Monitor the push in the ``#mdndev`` IRC channel. The final messages (one
-  for kuma, one for kumascript) are something like::
-
-    🎉 SUCCESS: Check Rollout Status: Branch prod-push build #7
```

Comment:
This could be replaced with:
```
  * Monitor the push in the ``#mdn-infra`` Slack channel. The final messages
    (one for kuma, one for kumascript) look like::

        🎉 SUCCESS Check Rollout Status
        🎉 SUCCESS Record Rollout
```